tools/Environments/DevHome.Environments/DevHome.Environments.csproj

```diff
@@ -22,6 +23,7 @@
     <PackageReference Include="Serilog.Sinks.Console" Version="5.0.1" />
     <PackageReference Include="Serilog.Sinks.Debug" Version="2.0.0" />
     <PackageReference Include="Serilog.Sinks.File" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Xaml.Behaviors.WinUI.Managed" Version="2.0.9" />

```

Comment:
This is already included in DevHome.Common (which is included in this project) so doesn't need to be re-included here.